var/spack/repos/builtin/packages/py-storm/package.py

```diff
@@ -11,6 +11,21 @@ class PyStorm(PythonPackage):
     homepage = "https://storm.canonical.com/"
     url      = "https://launchpad.net/storm/trunk/0.20/+download/storm-0.20.tar.gz"
 
+    version('0.23', sha256='01c59f1c898fb9891333abd65519ba2dd5f68623ac8e67b54932e99ce52593d3')
     version('0.20', sha256='0fa70043bb1a1c178c2f760db35f5956244cecf50dab7fb22d78be7507726603')
 
     depends_on('py-setuptools', type='build')
+    depends_on('py-six', when='@0.23:')
+    depends_on('python@2.7:2.8', when='@:0.20')
+    depends_on('python@2.7:2.8,3.5:', when='@0.21:')
+
+    def url_for_version(self, version):
+        url = self.url.split('/')
+        url[5] = '{0}'
+        if version >= Version('0.20'):
+            url[-1] = '/storm-{0}.tar.bz2'
+        else:
+            url[-1] = '/storm-{0}.tar.gz'
+        
+        url = '/'.join(url).format(version)
+        return url
```

Comment:
Why not just update `url`? 0.20 is the oldest version in Spack anyway. Moreover, if I don't have mixed anything up, all versions are available as `tar.bz2` according to https://launchpad.net/storm/+download.